src/purchases.ts

```diff
@@ -182,6 +182,7 @@ export default class Purchases {
      * @param {String} apiKey RevenueCat API Key. Needs to be a String
      * @param {String?} appUserID An optional unique id for identifying the user. Needs to be a string.
      * @param {boolean?} observerMode An optional boolean. Set this to TRUE if you have your own IAP implementation and want to use only RevenueCat's backend. Default is FALSE.
+     * @param {boolean?} usesStoreKit2IfAvailable An optional boolean. iOS-only. Set this to TRUE to enable StoreKit2 on compatible devices.
```

Comment:
we can apparently mark it as boolean by doing 
```suggestion
     * @param {boolean?} [usesStoreKit2IfAvailable=false] An optional boolean. iOS-only. Set this to TRUE to enable StoreKit2 on compatible devices.
```

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

We should update the others too, but they can all be done in a separate PR

Reply:
Oh cool! I just updated both to match that guideline (`boolean` instead of `boolean?`).